creatorkit/Glossary.md

```diff
@@ -0,0 +1,17 @@
+# Glossary
+
+**Creator Kit** - This is the name of the kit [available here](http://www.imgtec.com) that includes the hardware you need to complete the projects detailed in this guide.
+
+**Ci40** - Ci40 is largest board in your kit and is the IoT hub. It connects to the Internet, and has many connectivity options for constrained devices to connect to (WiFi/6LowPAN/BlueTooth). See full details of Ci40 on the [FlowM2M website](http://www.flowcloud.io)
+
+**IoT** - Abbreviation of "Internet of Things". IoT is a concept in which many devices are connected together to work more efficiently/effectively. Any system in which multiple devices are connected together and connected to the Internet can be considered to be an IoT system.
+
+**Contrained Device** - This term is a catch-all for very simple devices that connect to your Ci40 in a local network. Your Clicker boards included in the kit are an example of a Constrained Device.
+
+**OpenWRT** - This is the operating system used on Ci40 for the Creator Kit projects. Ci40 can run a variety of operating systems, but OpenWRT is used in these examples. [Openwrt website](https://openwrt.org/)
```

Comment:
It's not an operating system, OpenWrt is described as a Linux distribution for embedded devices.